about.html

```diff
@@ -1,73 +1,104 @@
-<!doctype html>
```

Comment:
The MVP for the about.html page was to wrap the existing content in semantic HTML, it appears you removed the contents of the file